packages/docs/src/pages/en/features/icon-fonts.md

```diff
@@ -11,34 +11,84 @@ related:
 
 # Icon Fonts
 
-Vuetify comes bootstrapped with support for Material Design Icons, Material Icons, Font Awesome 4 and Font Awesome 5. By default, applications will default to use [Material Design Icons](https://materialdesignicons.com/).
+Out of the box, Vuetify supports 4 popular icon font libraries—[Material Design Icons](https://materialdesignicons.com/), [Material Icons](https://material.io/resources/icons/?style=baseline), [Font Awesome 4](https://fontawesome.com/v4.7.0/) and [Font Awesome 5](https://fontawesome.com/).
 
 <entry-ad />
 
 ## Usage
 
-In order to change your font library, add the `iconfont` option during instantiation.
+To change your font library, import one of the pre-defined icon sets or provide your own.
 
 ```js
 // src/plugins/vuetify.js
 
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
+import { createVuetify } from 'vuetify'
+import { aliases, mdi } from 'vuetify/lib/iconsets/mdi'
 
-Vue.use(Vuetify)
+export default createVuetify({
+  icons: {
+    defaultSet: 'mdi',
+    aliases,
+    sets: {
+      mdi,
+    }
+  },
+})
+```
+
+```html
+<template>
+  <v-icon icon="mdi-home" />
+</template>
+```
+
+In the above examples we import the default `mdi` icon set and its corresponding aliases. These aliases reference commonly used types of icons that are utilized by Vuetify components.
+
+<alert type="info">
 
-export default new Vuetify({
+  While it is still possible to supply the icon value through the default slot in Vuetify 3.0 (`<v-icon>mdi-home</v-icon>`), we recommend using the `icon` prop instead.
+
+</alert>
+
+## Multiple icon sets
+
+Out of the box, Vuetify supports the use of multiple *different* icon sets at the same time. The following example demonstrates how to change the default icon font to Font Awesome (`fa`) while still maintaining access to the original Material Design Icons (`mdi`):
+
+```js
+// src/plugins/vuetify.js
+
+import { createVuetify } from 'vuetify'
+import { aliases, fa } from 'vuetify/lib/iconsets/fa'
+import { mdi } from 'vuetify/lib/iconsets/mdi'
+
+export default createVuetify({
   icons: {
-    iconfont: 'mdiSvg', // 'mdi' || 'mdiSvg' || 'md' || 'fa' || 'fa4' || 'faSvg'
+    defaultSet: 'fa',
+    aliases,
+    sets: {
+      fa,
+      mdi,
+    }
   },
 })
 ```
 
-Using a predefined option will pre-fill defaults based upon the selected preset. This will overwrite the defaults of components that have default **icon** values. For more information, view the default [icon preset values](https://github.com/vuetifyjs/vuetify/tree/master/packages/vuetify/src/services/icons/presets).
+```html
+<template>
+  <v-icon icon="fas fa-plus" /> // This renders a FontAwesome icon
+  <v-icon icon="mdi:mdi-minus" /> // This renders a MDI icon
+</template>
+```
+
+<alert type="info">
+
+  Note the `mdi:` prefix used in the *second* icon. It is necessary to provide the **correct** prefix when rendering icons that are **not** part of the default set.
```

Comment:
This seems kinda confusing, maybe say "It is not necessary to provide a prefix for icons from the `defaultSet`"